Orange/tests/test_xlsx_reader.py

```diff
@@ -48,6 +48,11 @@ def test_named_sheet(self):
         self.assertEqual(len(table.domain.attributes), 4)
         self.assertEqual(table.name, 'header_0_sheet-my_sheet')
 
+    def test_named_sheet_table(self):
+        table = Table(get_dataset("header_0_sheet.xlsx"), sheet="my_sheet")
```

Comment:
Usage implies `sheet` is an argument accepted in the constructor. Any future re-design needs to take this into consideration. Our Table constructor doesn't have a docstring. This is _not the usual_.